test/dlc_tests/conftest.py

```diff
@@ -89,6 +94,13 @@
     "inference",
 )
 
+# Immutable constant dictionary for nightly specific image fixtures
+# Each fixture must have a set of labels
+NIGHTLY_FIXTURES = {
+    "feature_smdebug_present": {"aws_framework_installed", "smdebug_installed"},
+    "feature_smddp_present": {"aws_framework_installed", "smddp_installed"},
+}
```

Comment:
Please also add `"feature_smmp_present"`, with label `"smmp_installed"` associated to it. I don't believe it needs `aws_framework_installed`.

Reply:
Please also add `"feature_aws_framework_present"`, with label `"aws_framework_installed"` associated to it.